algorithm/pge.metta

```diff
@@ -0,0 +1,421 @@
+! (register-module! ../../metta-moses)
+! (import! &self metta-moses:utilities:general-helpers)
+
+;; python bindings for random number generation and rounding
+! (bind! np (py-atom numpy))
+! (bind! rd (py-atom round))
```

Comment:
I would prefer if this name is round, It would be to understand what it does later